### ISSUE TITLE
Merging changes from SBOLInputDialog into SBOLDesigner

### ISSUE THREAD

**Issue by cjmyers** [SynBioDex/SBOLDesigner#199]
Within iBioSIm, gui/modelEditor/sbol/SBOLInputDialog.java, there is a branch of this class.  Merge this branch back into SBOLDesigner.

**Comment by michael13162** [comment]
@tramyn Do you mind walking me through what changes you made sometime?  SBOLInputDialog has 500 more lines than RootInputDialog, so it might be easier for me to just modify the original.

**Comment by tramyn** [comment]
@michael13162 Yes. Let's do this in person when we're both in lab. Let me know when.
